website_product_configurator/data/config_form_templates.xml

```diff
@@ -0,0 +1,994 @@
+                            t-att-data-old-val-id="(value_ids &amp; line.value_ids) and (value_ids &amp; line.value_ids).ids[0] or ''"
+                        >
+                            <t t-foreach="line.value_ids" t-as="value">
+                                <div class="radio-card-container">
+                                    <div class="info_config_attr_value_radio">
+                                        <span class="label_config_attr_value_radio">
+                                            <t t-esc="value.name" />
+                                        </span>
+                                        <t
+                                            t-set="price_extra"
+                                            t-value="extra_prices.get(value.id, 0)"
+                                        />
+                                        <t t-if="price_extra">
+                                            <span
+                                                class="label_config_price_extra_radio"
+                                            >
+                                                (+<t
+                                                    t-esc="price_extra"
+                                                    t-options="{'widget': 'monetary', 'display_currency': pricelist.currency_id, 'from_currency': website.currency_id}"
+                                                />)
+                                            </span>
+                                        </t>
+                                        <input
+                                            t-att-id="'%s%s' % (field_prefix, line.attribute_id.id)"
+                                            t-att-checked="'checked' if value.id in value_ids.ids else None"
+                                            t-att-type="line.multi and 'checkbox' or 'radio'"
+                                            t-att-class="'config_attr_value'"
+                                            t-att-value="value.id"
+                                            t-att-data-oe-id="value.id"
+                                            t-att-name="'%s%s' % (field_prefix, line.attribute_id.id)"
+                                            t-att-disabled="value.id not in available_value_ids and True or False"
+                                            t-att-title="value.name"
+                                        />
+                                    </div>
+                                    <label
+                                        class="image_config_attr_value_radio"
+                                        for="'%s%s' % (field_prefix, line.attribute_id.id)"
+                                        t-att-data-val-id="value.id"
+                                        t-attf-style="background-image:url(/web/image/product.attribute.value/{{value.id}}/image_medium);"
```

Comment:
Some initial testing, for the images the work you need:
t-attf-style="background-image:url(/web/image?model=product.attribute.value&amp;id={{value.id}}&amp;field=image);